add-tourpackage.php

```diff
@@ -322,6 +313,114 @@
+                                            if ($vehicleNumRows > 0) {
+                                                for ($a = 0; $a < $vehicleNumRows; $a++) {
+                                                    $vehicleData = $vehicleResultSet->fetch_assoc();
+                                            ?>
+                                                    <div class="col-12">
+                                                        <div class="row">
+                                                            <div class="col-2">
+                                                                <input type="checkbox" name="v-id" value="<?php echo $vehicleData['id']; ?>">
+                                                            </div>
+                                                            <div class="col-6">
+                                                                <span><?php echo $vehicleData["number"]; ?></span>
+                                                            </div>
+                                                            <div class="col-4">
+                                                                <span><?php echo $vehicleData["type"]; ?></span>
+                                                            </div>
+                                                        </div>
+                                                    </div>
+                                                <?php
+                                                }
+                                            } else {
+                                                ?>
+                                                <div class="col-12">
+                                                    <center>
+                                                        <span><i>No results found</i></span>
+                                                    </center>
+                                                </div>
+                                            <?php
+                                            }
+                                            ?>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="modal-footer">
+                                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+                                    <button type="button" class="btn btn-primary" onclick="getVehicleIDs();">Save changes</button>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <!-- vehicle selection model -->
+
+                    <!-- hotel selection model -->
+                    <div class="modal fade" id="hotel-selection-model" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
```

Comment:
Add this `data-bs-backdrop="static"` attribute to this modal. Your model still have chance to hide after click another area of your web page. You can give static behavior to model in this way.

Refer this : https://getbootstrap.com/docs/5.2/components/modal/#static-backdrop